lua/blink/compat/init.lua

```diff
@@ -24,4 +20,40 @@ function compat.setup(opts)
   end
 end
 
+local function setup_events()
+  local event = require('blink.compat.event')
+
+  local function make_entry(item)
+    return {
+      entry = { completion_item = item, source = item._source or {} },
+    }
+  end
+
+  local original_accept = require('blink.cmp.accept')
+  local function new_accept(item)
+    original_accept(item)
+    event:emit('confirm_done', make_entry(item))
+  end
+  package.loaded['blink.cmp.accept'] = new_accept
+
+  local autocomplete = require('blink.cmp.windows')
```

Comment:
Right now it crashes with module not found. I think this was meant instead.

```suggestion
  local autocomplete = require('blink.cmp.windows.autocomplete')
```

Reply:
Otherwise it seems to load fine. Although the codeium's completions don't update in the blink window once it has opened. So only the first version is shown until a separator (`.`, ` `, ...) is reached.

Reply:
I've fixed the crash. Note the change in spec - blink.compat must not be in blink.cmp's dependencies, otherwise `autocomplete` won't be loaded correctly.

As for the second problem, it's probably because `codeium` doesn't set `isIncomplete = true` on its completion response. I'll look into following `nvim-cmp` more closely for that case.

If https://github.com/Saghen/blink.cmp/issues/261 gets resolved, then you could also fix this by setting
```lua
transform_response = function(_, response)
  response.is_incomplete_forward = true
  return response
end
```
on the provider